src/multiview_stitcher/vis_utils.py

```diff
@@ -21,6 +21,10 @@ def plot_positions(
     n_colors=2,
     nscoord=None,
     display_view_indices=True,
+    custom_labels=None,
+    label_size=10,
```

Comment:
Agreed this is convenient! Could we rename these to "view_labels" and "view_labels_size"? I feel this makes it slightly more clear.

Reply:
Absolutely, will change labels_size to view_labels_size and push another commit. Any other suggestions, please let me know, and also feel free to modify anything that makes more sense. :)

Reply:
Perfect, thank you! If you agree, I'd also propose to change `custom_labels` to `view_labels` for consistency.

Reply:
Even better, change pushed.

Reply:
Thank you!